cypress/page/menu-content.page.ts

```diff
@@ -3,7 +3,7 @@ class MenuContentPage {
   private menuContentPageURL: string
 
   constructor() {
-    this.tShirtMenu = '#block_top_menu > ul > li:nth-child(3) > a';
+    this.tShirtMenu = '#block_top_menu > ul > li > [href*="id_category=5"]';
```

Comment:
good catch, what about if, instead of using the href, you use the title with the a tag:
`#block_top_menu > ul > li > a[title="T-shirts"]`